require-config.js

```diff
@@ -0,0 +1,7 @@
+require.config({
```

Comment:
This is the RequireJS config file. It tells require where to find 'chai' and 'knockout'. It also specifies what file should be run when the page loads (in this case 'lib/app.js').